### PR TITLE
[Hacktoberfest] PHP IDE feature 

### PR DESCRIPTION
**Pull request by vitorf7**
According to this article https://www.ubuntupit.com/how-to-install-and-use-php-composer-on-linux-distributions/
Here's what they say for PHP in arch Linux if you wanna test Chris:
```shell
sudo pacman -Syu php
curl -sS https://getcomposer.org/installer | php
sudo pacman -S composer
sudo mv composer.phar /usr/local/bin/composer
```
Next clone a simple code kata repo I had from a long time ago: `git clone https://github.com/vitorf7/code-katas`
Then:
```shell
cd code-katas
composer install
```
This should be enough to start testing.

---

Tackles #20

### CODE REVIEW

File: config.lua [hunk 1]
```diff
@@ -0,0 +1,63 @@
+-- ===================================== General Section ============================================
+lvim.log.level = "warn"
```
Comment (ChristianChiarulli) [inline config.lua]: remove all the extra settings here that don't pertain to the language

File: config.lua [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+		name = "Listen for Xdebug",
+		port = 9000,
+	},
+}
+
+-- ======================================= Dap Section ==============================================
+lvim.builtin.dap.active = true
+
+-- Key Maps
+lvim.builtin.which_key.vmappings["L"] = {
```
Comment (ChristianChiarulli) [inline config.lua]: we have all of these available in `d`

**Review by ChristianChiarulli — CHANGES_REQUESTED** [review]

File: config.lua (hunk 1 above)
Comment (vitorf7) [reply config.lua]: @ChristianChiarulli Thanks, I have removed it

File: config.lua (hunk 2 above)
Comment (vitorf7) [reply config.lua]: Yep, removed this and have re-organised the file.
May need someone to sanity check that this works correctly

**Pull request merged**